apps/web/src/components/Feed/Socialize/RemainingCommentCount.tsx

```diff
@@ -43,13 +45,17 @@ export function RemainingCommentCount({
     queryRef
   );
 
+  const { query: routerQuery } = useRouter();
+
   const openCommentsModal = useCommentsModal({ eventRef: event, queryRef: query });
 
   if (totalComments <= 1) {
     return null;
   }
 
-  return (
+  return routerQuery.displayMode === CommunityPageDisplayMode.PRESENTATION ? (
```

Comment:
supernit: something like this could probably be a nice util in the future:

```ts
useRouteQuery(key, value): boolean

// e.g.

const isPresentationMode = useRouteQuery('displayMode', CommunityPageDisplayMode.PRESENTATION)
```